src/jsifier.js

```diff
@@ -230,6 +230,13 @@ function ${name}(${args}) {
         return;
       }
 
+      if (symbolsOnly) {
+        if (!isJsOnlyIdentifier(ident) && LibraryManager.library.hasOwnProperty(ident)) {
+          librarySymbols.push(ident);
+        }
+        return;
+      }
+
```

Comment:
It's not obvious to me why this is equivalent. First, the WASM_EXPORTS check below can early exit before the `librarySymbols.push` - don't we need to check that still? Second, there are more early returns, like new line 256, so the results seem like they could differ compared to before?

Reply:
WASM_EXPORTS is not set when `--symbol-only` is used since that is part of the output the linker.   And the linker is only run after `--symbol-only`.

Reply:
The first early return there is in the `!LibraryManager.library.hasOwnProperty(ident))` block..  and you can see that we when ident is not part of the library we also do nothing and early return here.

Reply:
I will confirm the number of symbols is the same both before after this change.